requirements.txt

```diff
@@ -1,34 +1 @@
-aniso8601==3.0.2
-astroid==2.0.4
-atomicwrites==1.2.1
-attrs==18.2.0
-certifi==2018.10.15
-chardet==3.0.4
-Click==7.0
-coverage==4.5.1
-coveralls==1.5.1
-docopt==0.6.2
-Flask==1.0.2
-Flask-JWT-Extended==3.13.1
-Flask-RESTful==0.3.6
-gunicorn==19.9.0
-idna==2.7
-isort==4.3.4
-itsdangerous==0.24
-Jinja2==2.10
-lazy-object-proxy==1.3.1
-MarkupSafe==1.0
-mccabe==0.6.1
-more-itertools==4.3.0
-pluggy==0.7.1
-py==1.7.0
-PyJWT==1.6.4
-pylint==2.1.1
-pytest==3.8.2
-pytz==2018.5
-requests==2.20.0
-six==1.11.0
-typed-ast==1.1.0
-urllib3==1.24
-Werkzeug==0.14.1
-wrapt==1.10.11
+psycopg2==2.7.5
```

Comment:
Why did you remove the rest of the packages?

Reply:
Fixed